deltachat-ffi/deltachat.h

```diff
@@ -2588,6 +2594,85 @@ char* dc_get_last_error (dc_context_t* context);
 void dc_str_unref (char* str);
 
 
+/**
+ * @class dc_backup_sender_t
+ *
+ * Set up another device.
+ */
+
+/**
+ * Create an object for sending a backup to another device.
+ * The backup is sent through an peer-to-peer channel which is bootstrapped by a QR-code.
+ * As a backup contains the whole state of the account, including credentials,
+ * this can be used to setup a new device.
+ *
+ * @memberof dc_backup_sender_t
+ * @param context The context.
+ * @param folder Path to create the backup in before it is sent out.
+ *     The path must exist and being accessible at least until the other device has received all data.
+ *     The folder is not cleaned up by the backup send;
+ *     a temporary directory seems to be good place therefore.
+ * @param passphrase Used to at-rest-encrypt the backuped database in `folder` before sending,
+ *     similar to dc_imex().
+ *     The same passphrase must be given to dc_receive_backup() on the other device.
+ *     If NULL or empty string is given, the sent backup is not encrypted at rest.
+ *     The transport of the backup is always encrypted additionally.
```

Comment:
If the transport is encrypted why is this passphrase still needed?  Would be nice if this described what it is protecting against as well.

Reply:
This passphrase is there to encrypt the sql database. It is the same as the existing other backup methods. I personally would love to drop it, but didn't want to open that can of worms.

Reply:
cc @r10s

Reply:
yeah, i'd also better leave that can of worms closed, at least for now :)